src/components/cards/ThankYouCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { Card } from "./Card";
+import styles from "./ThankYouCard.module.scss";
+
+export default function ThankYouCard() {
+  return (
+    <Card className={styles.card}>
+      <svg
```

Comment:
I would create separate file for this svg and just import it into component, so it would be way more clean